Leanplum-SDK/Classes/Managers/Networking/LeanplumRequest.m

```diff
@@ -279,19 +216,19 @@ - (void)sendIfConnectedSync:(BOOL)sync
 
 - (void)attachApiKeys:(NSMutableDictionary *)dict
 {
-    dict[LP_PARAM_APP_ID] = appId;
-    dict[LP_PARAM_CLIENT_KEY] = accessKey;
+    dict[LP_PARAM_APP_ID] = [LPAPIConfig sharedConfig].appId;
+    dict[LP_PARAM_CLIENT_KEY] = [LPAPIConfig sharedConfig].accessKey;
```

Comment:
Could this be a part of `LPAPIConfig` ? Then `LP_PARAM_APP_ID` only needs to live in `LPAPIConfig` and it can also simplify LeanplumSocket.m
Not trying to scope creep but I think it'd be easy, worth it, and more DRY.